setup.py

```diff
@@ -73,7 +73,7 @@ def package_data(pkg, root_list):
     name='xblock-problem-builder',
     version='2.7.2',
     description='XBlock - Problem Builder',
-    packages=['problem_builder', 'problem_builder.v1'],
+    packages=['problem_builder', 'problem_builder.v1', 'problem_builder.management', 'problem_builder.management.commands'],
```

Comment:
Tip: It's even better to just [use `find_packages()`](https://github.com/open-craft/xblock-group-project-v2/commit/2d4406499cdda314e03d329991244cb0b7e974ae#diff-2eeaed663bd0d25b7e608891384b7298)